ups/eupspkg.cfg.sh

```diff
@@ -1,7 +1,56 @@
-export SCONSFLAGS=$SCONSFLAGS" USE_UNKNOWN_VARS=true TMV_DIR="$TMV_DIR" PREFIX="$PREFIX" PYPREFIX="$PREFIX"/lib/python EXTRA_LIB_PATH="$TMV_DIR"/lib EXTRA_INCLUDE_PATH="$TMV_DIR"/include"
+export SCONSFLAGS=$SCONSFLAGS" USE_UNKNOWN_VARS=true TMV_DIR="$TMV_DIR\
+" PREFIX="$PREFIX" PYPREFIX="$PREFIX"/lib/python EXTRA_LIB_PATH="$TMV_DIR\
+"/lib EXTRA_INCLUDE_PATH="$TMV_DIR"/include"
 
 pathToPython=$(which python)
-export DYLD_FALLBACK_LIBRARY_PATH="${pathToPython%bin/python}/lib"
+pathToPythonLib="${pathToPython%bin/python}/lib"
+
+build(){
+
+    # If running on a Mac, use install_name_tool to rewrite
+    # the loading addresses for libpython*dylib and
+    # libboost_python.dylib to make
```

Comment:
This whole approach gives me the heebie-jeebies. Modifying previously installed libraries sounds a bit dangerous.